docassemble/ALKilnInThePlayground/data/questions/run_alkiln_tests.yml

```diff
@@ -2159,83 +2552,428 @@ content: |
+    import json
+    import traceback
+
+    message = ""
+    result = {}
+
+    # config_path_result = get_config_path()
+    # if not config_path_result["ok"]:
+    #   return { "ok": False, "message": not config_path_result["message"], "result": result }
+    # local_config_path = config_path_result["result"]
+
+    try:
+      with open( json_filepath, 'r', encoding='utf-8') as json_file:
+        try:
+          result = json.load( json_file )
+      
+        except Exception as json_load_error:
+          message = f"""Error loading json from file at "{ json_filepath }":\n{ "\n".join(traceback.format_exception( json_load_error )) }"""
+          _log(f"""🔎 ALKiln WARNING ALKP0013: { message }""")
+          return {
+            "ok": False, "message": message, "result": result
+          }
+    except Exception as json_open_error:
+      message = f"""Skipped opening json file at "{ json_filepath }":\n{ "\n".join(traceback.format_exception( json_open_error )) }"""
+      _debug(f"""🖊️ ALKiln NOTE: { message }""")
+      return {
+        "ok": False, "message": message, "result": result
+      }
+
+        # try:
+        #   result = {}
+        #   json.dump(result, config_file, ensure_ascii=False, indent=2)
+  
+        # except Exception as config_dump_error:
+        #   message = f"""Error dumping json to config file at "{ local_config_path }":\n{ "\n".join(traceback.format_exception( config_dump_error )) }"""
+        #   _log(f"""🤕 ALKiln ERROR ALKP0015: { message }""")
+        #   return {
+        #     "ok": False, "message": message, "result": result
+        #   }
+  
+    return { "ok": True, "message": message, "result": result }
+---
+reconsider: True
+need:
+  - get_config_path
+  # - get_existing_path
+code: |
+  """
+  Return the runtime_config.json path or, if it doesn't yet exist, None
+  """
+  # temp_config_path = get_existing_path( "runtime_config.json" )
+  # if temp_config_path is None:
+  #   config_path_problem = 'No "runtime_config.json" file exists.'
+  #   _log(f'🤕 ALKiln ERROR: { config_path_problem }')
+  # config_path = temp_config_path
+  config_path = get_config_path()["result"]
 ---
 need:
   - get_existing_path
 code: |
-  temp_config_path = get_existing_path( "runtime_config.json" )
-  if temp_config_path is None:
-    config_path_problem = 'No "runtime_config.json" file exists.'
-    _log(f'🤕 ALKiln ERROR: { config_path_problem }')
-  config_path = temp_config_path
+  def get_config_path():
+    """
+    Return the runtime_config.json path or, if it doesn't yet exist, None
+    """
+    ok = True
+    message = ""
+    result = get_existing_path( "runtime_config.json" )
+  
+    if result is None:
+      ok = False
+      message = 'No "runtime_config.json" file exists.'
+      _log(f'🔎 ALKiln WARNING: { message }')
+    return { "ok": ok, "message": message, "result": result }
 ---
 id: default config_path_problem
 code: |
-  config_path_problem = ''
+  config_path_problem = ''  # Default
```

Comment:
I believe this is no longer used. It might be useful to use it in `show_output`, though, especially if we can improve the message.